docs/content/preview/yugabyte-platform/back-up-restore-universes/schedule-data-backups/ycql.md

```diff
@@ -34,44 +34,39 @@ You can use YugabyteDB Anywhere to perform regularly scheduled backups of Yugaby
 
-1. Navigate to **Universes**.
+- Navigate to **Universes**.
+- Select the name of the universe for which you want to schedule backups.
+- Select the **Tables** tab and click **Actions** to verify that backups are enabled. If disabled, click **Enable Backup**.
+- Select the **Backups** tab and then select **Scheduled Backup Policies**. 
+- Click **Create Scheduled Backup Policy** to open the dialog shown in the following illustration:
 
-1. Select the name of the universe for which you want to schedule backups.
+<br><br>
 
-1. Click the **Tables** tab and verify that backups are enabled. If disabled, click **Enable Backup**.
+![Create Backup form](/images/yp/scheduled-backup-ycql.png)<br><br>
 
-1. Select the **Backups** tab and then click **Create Scheduled Backup** to open the **Create Backup** dialog shown in the following illustration:
-    <br/>
-    <br/>
+- Provide the backup policy name.
+- Specify the interval between backups or select **Use cron expression (UTC)**.
+- Set the API type as YCQL. 
+- Select the backup storage configuration. Notice that the contents of the **Select the storage config you want to use for your backup** list depends on your existing backup storage configurations. For more information, see [Configure backup storage](../../configure-backup-storage/).
+- Select the database to backup, and then specify whether you want to back up all tables in the keyspace or only specific tables.
+- Specify the period of time during which the backup is to be retained. Note that there's an option to never delete the backup.
+- Specify the number of threads that should be available for the backup process.
+- Click **Create**.
 
-    ![Create Backup form](/images/yp/scheduled-backup-ycql.png)<br>,br>
-
-1. Enter the **Backup frequency** (interval in milliseconds) or a **Cron expression (UTC)***. For details on valid `cron` expression formats, hover over the question mark (`?`) icon.
-
-1. Select the **YCQL** tab and enter values for the following fields:
-
-    - **Storage**: Select the storage type: `GCS Storage`, `S3 Storage`, or `NFS Storage`.
-    - **Keyspace**: Select your keyspace from the drop-down list of predefined keyspaces.
-    - **Tables to backup**: Select either **All Tables in Keyspace** or the specific tables to back up.
-    - **Parallel Threads**: Enter or select the number of threads. The default is `8`.
-    - **Number of Days to Retain Backup**: Default is unspecified which means to retain indefinitely.
-
-1. Click **OK**. <br>
-
-    The initial backup begins immediately.
-
-Subsequent backups are created based on the value you specified for **Backup frequency** or **Cron expression**.
+Subsequent backups are created based on the value you specified for **Set backup intervals** or **Use cron expression**.
 
 ## Disable scheduled backups
 
-You can temporarily disable all scheduled backups as follows:
+You can temporarily disable all scheduled backups, as follows:
```

Comment:
This step disables all kinds of backups on the universe. There is way to enable/disable schedule policy on each schedule defined